CHANGELOG.md

```diff
@@ -2,6 +2,24 @@ faucet-pipeline-core version history
 ====================================
 
 
+v2.0.1
+------
+
+_2025-02-01_
+
+notable changes for end users:
+
+* dropped support for obsolete Node versions; now requires Node v18 or later
+* reduced number of dependencies
```

Comment:
This sounds far too mundane: It's a significant improvement IMO, but ... 🤷

Reply:
I would keep it. Most people won't care, but I think it is still worth putting four words in the changelog.

Reply:
My concern was more that this sounds like a boring line item when it's really "yay, look at what @moonglum figured out!"...

Reply:
Thanks, appreciated :bow: